utils/helm/speckle-server/templates/deployment-frontend.yml

```diff
@@ -43,3 +43,7 @@ spec:
             port: 80
           initialDelaySeconds: 5
           periodSeconds: 5
+        
+        env: 
+          - name: FILE_SIZE_LIMIT_MB
```

Comment:
Does the following line also need to be changed to use this environment variable? https://github.com/specklesystems/speckle-server/blob/a10c49e7310ce84cbbbb9657a8b73514336214cb/packages/frontend/src/main/components/common/file-upload/FileUploadZone.vue#L84

Reply:
Right, this is missed.

Reply:
this is the bit I was struggling with, I didn't know how Vue was able to get environment variables from the server.  Would like an explanation if you know/figure it out!

Reply:
you cannot, afaik in static build vue. It needs to come from the server via an API. That is also cleaner anyhow, so thats what im going to add.